src/firewall/command.py

```diff
@@ -43,31 +43,35 @@ def set_verbose(self, flag):
     def get_verbose(self):
         return self.verbose
 
-    def print_msg(self, msg=None):
-        if msg is not None and not self.quiet:
-            sys.stdout.write(msg + "\n")
-
-    def print_error_msg(self, msg=None):
-        if msg is not None and not self.quiet:
-            sys.stderr.write(msg + "\n")
-
-    def print_warning(self, msg=None):
-        FAIL = "\033[91m"
-        END = "\033[00m"
-        if sys.stderr.isatty():
+    def print_msg(self, msg=None, *, force=False, file=None, exit_code=None):
```

Comment:
AFACIS, `force` is not used in this PR or the PR for lockdown removal. Why is it being added? Same for `file`.

IMO, this is change for change's sake.

Reply:
ah, force was supposed to be used by the lockdown removal. Will add it there.

That print by default honors `self.quiet` is at least questionable, because quiet mode should not mean that all messages are suppressed, but rather that certain verbose output is suppressed.

Regardless of whether `force=False` is good to be the default value. The message that the Lockdown command does nothing, is not something that should be suppressed by `--quiet`.

```bash
$ ./src/firewall-cmd --lockdown-on -q
Lockdown feature is deprecated and removed.
```